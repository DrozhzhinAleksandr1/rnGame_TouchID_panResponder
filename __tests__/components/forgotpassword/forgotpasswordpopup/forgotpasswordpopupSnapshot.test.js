import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {ForgotPasswordPopUp}  from '../../../../app/components/forgotpassword/forgotpasswordpopup/forgotpasswordpopup';

it('create snapshot ForgotPasswordPopUp', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<ForgotPasswordPopUp />);
    expect(toolbarSnapshot).toMatchSnapshot();
});
