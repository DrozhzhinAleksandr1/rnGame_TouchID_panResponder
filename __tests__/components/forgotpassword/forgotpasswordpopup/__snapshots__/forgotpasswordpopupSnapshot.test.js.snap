// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot ForgotPasswordPopUp 1`] = `null`;
